docs/kwik-format.md

```diff
@@ -34,8 +33,8 @@ All files are in HDF5.
           * information about cluster groups
           * events, event_types
           * aesthetic information, user data, application data
-      * the **KWX** file contains the **spiking data**: features, masks, waveforms
-      * the **KWD** files contain the **raw/filtered recordings**
+      * the **KWX** file contains the **spiking data**: features, masks
+      * the **KWD** files contains the **raw/filtered recordings**
```

Comment:
the KWD file is "sort of" deprecated in phy... by default we use `.dat`, although both are supported... but users should slowly move away from KWD...